website/blog/2022-08-08-benchmark-tpc-h.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Benchmarking Databend using TPC-H
+description: Benchmarking Databend using TPC-H
+slug: benchmark
+date: 2022-08-08
+tags: [databend, benchmark, TPC-H]
+authors:
+- name: BohuTANG
+  url: https://github.com/BohuTANG
+  image_url: https://github.com/BohuTANG.png
+---
+
+This post gives you a general idea about the TPC-H benchmark and explains how to run a TPC-H benchmark on Databend.
+
+## What's TPC-H?
+
+TPC-H is a decision support benchmark. It consists of a suite of business-oriented ad hoc queries and concurrent data modifications. The queries and the data populating the database have been chosen to have broad industry-wide relevance. This benchmark illustrates decision support systems that examine large volumes of data, execute queries with a high degree of complexity, and give answers to critical business questions.
+
+The TPC-H benchmark simulates a system for online sales of parts and components and defines eight tables in total. The structure, data volume, and mutual relationship of each table are shown in the figure below:
+
+![](../static/img/blog/tables.jpeg)
+
+The benchmark workload consists of twenty-two decision support queries that must be executed as part of the TPC-H benchmark. Each TPC-H query asks a business question and includes the corresponding query to answer the question. More information about TPC-H can be found at https://www.tpc.org/tpch/.
+
+## Running TPC-H Benchmark on Databend
+
+This section describes the steps to run the TPC-H benchmark on Databend and provides the related scripts.
+
+### Step 1: Generate test data with TPC-H Docker
+
+The following code pulls a docker image and allocates the data in the path where you are running the TPC-H benchmark.
+
+```shell
```

Comment:
It would be better to introduce the `-s`, which introduced in: https://github.com/databloom-ai/TPCH-Docker

Here `-s 1` is for example, users can set it to larger to do benchmark.